feat: Release `1.0.0-pre.9` (#3579)
CloudFire-LLC/cloudfire: website/src/components/FAQ/readme.mdx

Service accounts facilitate secure connections from a server, VM or container to
a Resource. Because service accounts do not involve a user, they can’t fulfill
2FA requirements, and use the Firezone Client in a non-interactive way (headless

> mode).

Reviewer: Github isn't letting me add a suggestion here but

This could parse as "Because services accounts do not involve a user, they can't (fulfill requirements, and use the Firezone Client in a non-interactive way)"

Consider "Because service accounts do not involve a user, they can't (fulfill 2FA requirements), so they (use the Firezone Client in a non-interactive way)"

Or to put the positive first and details second, "Service accounts use the Firezone Client in non-interactive mode ("headless" mode), since they do not involve user interaction and cannot fulfill 2FA requirements."

## Performance

#### Does all my traffic go through Firezone?

> 

Reviewer: ```suggestion
When Clients connect to anything outside your Firezone network, they do not go through Firezone. They are resolved with the upstream DNS or system DNS, and the OS sends traffic over the other network interfaces, ignoring Firezone.
```

Network traffic is always end-to-end encrypted, and by default, routes directly

> to Gateways running on your infrastructure. In rare circumstances, encrypted
> traffic can pass through our global relay network if a direct connection cannot
> be established. Firezone can never decrypt the contents of your traffic.

Reviewer: ```suggestion
to Gateways running on your infrastructure. If we can't establish a direct connection,
encrypted traffic may pass through our global relay network. 
Firezone can never decrypt the contents of your traffic.
```

(Unless we get an NSA letter ofc)

#### How do end users get Firezone?

Windows and Linux users can download the Client from
[here](https://github.com/firezone/firezone/releases). MacOS, iOS, iPadOS,
Android, and ChromeOS users can download the Client from their device’s app
store.

#### Do users need to interact with the Firezone Client?

> The client app is designed to be unobtrusive, and once a user clicks connect and
> authenticates with their credentials, it should run in the background without

Reviewer: ```suggestion
The client app is designed to be unobtrusive, and once a user signs in with their credentials, it should run in the background without
```

There isn't a "Connect" button, is there? On Windows it says "Sign In"

Firezone helps improve organizations’ cybersecurity posture by offering a modern

> approach to securing access to sensitive Resources in their private network.

Reviewer: nit: I get "Gateway" and "Client" if it's short for "Firezone Gateway" and "Firezone Client" but this isn't really a proper noun, right?

```suggestion
approach to securing access to sensitive resources in their private network.
```

Author: Yeah I was trying to determine how to think about that. In Firezone context it's a proper noun, but since here we preface it with "sensitive" then I think that makes it a generic noun.
